node-tests/tag-include-posts.js

```diff
@@ -0,0 +1,108 @@
+const { expect } = require('chai');
+
+const { createBuilder, createTempDir } = require('broccoli-test-helper');
+
+const TagIncludePosts = require('../lib/tag-include-posts');
+
+let output;
+let input;
+
+async function buildFiles(files) {
+  input.write(files);
+
+  const subject = new TagIncludePosts(input.path());
+  output = createBuilder(subject);
+
+  await output.build();
+
+  return output.read();
+}
+
+describe('Tag Generator', function() {
+  beforeEach(async () => {
+    input = await createTempDir();
+  });
+
+  afterEach(async () => {
+    try {
+      await input.dispose();
+    } finally {
+      // do nothing
```

Comment:
Probs same as not including this try/finally? Did we need a catch for any reason?

Reply:
I'm including a bigger/better comment that explains why we want to do it this way 👍